Intent2.0 (#1)
IntentSoftware/Docs: articles/tutorials/index.md

# Tutorials

> We try to help you understand Intent Architect as a newcomer by providing a couple of tutorials to work through.

Author: ```suggestion
This is a collection of tutorials which can be used to help you learn how to use Intent Architect.
```

> Please choose the one that closely relates to the feature you're trying to learn.

Author: ```suggestion
Please choose from one of the available tutorials below.
```

> ### [Getting Started - Create an ASP.NET Core web app](create_an_aspnetcore_web_app.md)

Author: Let's change this to an xref with a blank titles so it will automatically populate the title from the article.

> Let us show you how easy it is to create a new ASP.NET Core Web Application using Intent Architect and have it up and running in 5 minutes.

Author: ```suggestion
Learn how to use Intent Architect and our pre-made to Modules to create and manage an ASP.NET Core Web Application. You will have your application up and running in less than 5 minutes.
```

> ### [Getting Started - Create your own Module](create_your_own_module.md)

Author: Let's change this to an xref with a blank titles so it will automatically populate the title from the article.

> Once you have understood that one of the powerful features of Intent Architect comes from selecting modules, you will love the fact that you can create your own. This tutorial will take you through the process of creating your own ASP.NET Web API based code generation module.

Author: ```suggestion
Intent Architect's true power stems from making it easy for developers template and patternise their own custom code patterns and architecture. This tutorial will take you through the process of creating your own ASP.NET Web API based code generation module.
```
